### DISCUSSION TITLE
ext dont work

### DISCUSSION THREAD

**Discussion by GlizxyBoi** [3kh0/ext-remover#278] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
it dont work anympore pls fix
![Screenshot 2022-11-21 11 10 11 AM](https://user-images.githubusercontent.com/94561504/203117733-9ff28ac3-612a-4d03-b6cf-7ec2a6b0a757.png)

**Comment by 1-Mythical** [comment]
Screenshot not working

**Reply by GlizxyBoi** [reply]
just click on it\

**Comment by GlizxyBoi** [comment]
help pls fix
![Screenshot 2022-11-21 11 10 11 AM](https://user-images.githubusercontent.com/94561504/203124827-392ac09f-2f53-4825-8dc4-6c72b253db44.png)

**Reply by ghost** [reply]
does't it work it says it works with anything because i have blocksi

**Reply by GlizxyBoi** [reply]
> does't it work it says it works with anything because i have blocksi

no it don't work anymore

**Reply by ghost** [reply]
kid understand this it's doesn't work this dude is trying to scam you and wasting your time all you need is zip file

**Comment by ILikeBypassing** [comment]
what version are you on? 106 and above is PATCHED

**Reply by GlizxyBoi** [reply]
can yall make it so it supports newer versions

**Reply by skoopy-star** [reply]
it takes time

**Reply by GlizxyBoi** [reply]
> what version are you on? 106 and above is PATCHED

can u fix so it works on this version and above

**Comment by GlizxyBoi** [comment]
my school took my comp and i found out they updated it

**Reply by skoopy-star** [reply]
you have to wait for them to fix read other discussions

**Comment by GlizxyBoi** [comment]
ight update me when they fix it

**Reply by Theblackmamba824** [reply]
1 of the devs said it might take a month

**Reply by skoopy-star** [reply]
the dev said that 1 month ago

**Reply by Theblackmamba824** [reply]
no but 1 of them just said that about pingot their new project

**Reply by GlizxyBoi** [reply]
> 1 of the devs said it might take a month

fuck i can deal with that ig

**Reply by GlizxyBoi** [reply]
what the hell is pingot

**Reply by GlizxyBoi** [reply]
> no but 1 of them just said that about pingot their new project

whats pingot

**Comment by bartholemuew** [comment]
bro is lagging fr fr how did you just notice it got patched

**Reply by GlizxyBoi** [reply]
> bro is lagging fr fr how did you just notice it got patched

bro how am i lagging i updated my comp and it got patched

**Reply by synapt0** [reply]
how have you not known about this issue dawg u laggin ong

**Reply by SomePVZPlayerMatt** [reply]
he stuck in time

**Reply by bartholemuew** [reply]
fr fr bro probably still in october 💀

**Reply by obkduishinijf** [reply]
bro stuck in 2021 💀

**Comment by GlizxyBoi** [comment]
bro i knew abt it when i updated my comp

**Comment by GlizxyBoi** [comment]
how tf am i stuck in time u fool

**Reply by 3kh0** [reply]
wa

**Reply by nicolasog12** [reply]
idfk

**Comment by GlizxyBoi** [comment]
bro im in november u fool my school didnt give us the update till november
u dick

**Reply by bartholemuew** [reply]
nvm bro mb mb

you in february 💀

**Reply by GlizxyBoi** [reply]
> nvm bro mb mb
> 
> you in february 💀

bro how dumb are u bro ur school probably gave u the update before my school like bro stfu im in november

**Reply by bartholemuew** [reply]
bro in 1776 🗿

**Reply by 3kh0** [reply]
🗿

**Reply by GlizxyBoi** [reply]
> bro in 1776 🗿

how do u think I am

**Reply by GlizxyBoi** [reply]
> bro in 1776 🗿

i swear u r the dumbest person ever

**Reply by bartholemuew** [reply]
he 100% in 1776 bro didnt deny it

**Reply by GlizxyBoi** [reply]
> he 100% in 1776 bro didnt deny it

i didn't admit it and bro I'm in 2022 fool and bro ur name is bartholemuew

**Reply by nicolasog12** [reply]
> bro how dumb are u bro ur school probably gave u the update before my school like bro stfu im in november

bro cant take a joke 💀☠

**Reply by bartholemuew** [reply]
> > bro how dumb are u bro ur school probably gave u the update before my school like bro stfu im in november
> 
> bro cant take a joke 💀☠

he def in 1992 fr ong

**Reply by obkduishinijf** [reply]
> > > bro how dumb are u bro ur school probably gave u the update before my school like bro stfu im in november
> > 
> > 
> > bro cant take a joke 💀☠
> 
> he def in 1992 fr ong

frfr

**Comment by Aka-but-nice** [comment]
First of all, yall watch too much flash, and second of all, for making the bookmarklet how' 'bout we try to make it where once you click the bookmarklet it loads up the GUI, but instead of making the device thinking that it is the webstore lets try to make it think your policy or admin!

**Reply by bartholemuew** [reply]
tell me you have no idea what you're talking about without telling me you have no idea what you're talking about

**Reply by GlizxyBoi** [reply]
> First of all, yall watch too much flash, and second of all, for making the bookmarklet how' 'bout we try to make it where once you click the bookmarklet it loads up the GUI, but instead of making the device thinking that it is the webstore lets try to make it think your policy or admin!

my guy ive only seen one flash movie

**Comment by nicolasog12** [comment]
**Wtf is going on** anyways kid...
If you use v106 or ABOVE on your chromebook it has been _PATCHED._ Hopefully there will be a fix but if there ISN'T then _ON GOD_ just use a _PROXY._
https://all-star-math.ml
https://all-star-ela.ml

**Comment by GlizxyBoi** [comment]
ok can yall make nearpod cheats like a js and others???? if u can thx and also how is it coming on the new ext disabler

**Reply by skoopy-star** [reply]
skid

**Reply by GlizxyBoi** [reply]
> skid

what

**Comment by GlizxyBoi** [comment]
yoooo can yalll fix this pls i cant keep using the extra computers anymore its kinda obvious what im doing

**Reply by Theblackmamba824** [reply]
https://profuse-frill-roarer.glitch.me/

**Reply by Theblackmamba824** [reply]
be patient damn its not like its an ez thing 2 fix ive been waiting seens September

**Reply by GlizxyBoi** [reply]
> be patient damn its not like its an ez thing 2 fix ive been waiting seens September

dam

**Comment by Aka-but-nice** [comment]
Refer to #347
